packages/angular/src/generators/ng-add/migrate-from-angular-cli.ts

```diff
@@ -105,21 +119,14 @@ export async function migrateFromAngularCli(
       updateRootEsLintConfig(tree, eslintConfig, options.unitTestRunner);
       cleanupEsLintPackages(tree);
     }
-
-    await formatFiles(tree);
   }
 
-  if (defaultProject) {
-    const workspaceConfig = readWorkspaceConfiguration(tree);
-    updateWorkspaceConfiguration(tree, {
-      ...workspaceConfig,
-      defaultProject: defaultProject.name,
-    });
-  }
+  deleteAngularJson(tree);
 
   if (!options.skipInstall) {
     return () => {
       installPackagesTask(tree);
+      formatFilesTask(tree);
```

Comment:
Why change to using a task?

Reply:
Because `prettier` only gets installed in the line above, in a task. Formatting doesn't work before that.

Reply:
I see.. what if we don't add `prettier`?

Reply:
Okay then. :thinking: 

Let's discuss this more after this PR lands.